heron/spi/src/java/com/twitter/heron/spi/common/keys.yaml

```diff
@@ -62,6 +62,12 @@ SCHEDULER_IS_SERVICE:                heron.scheduler.is.service
 SCHEDULER_JAR:                       heron.jars.scheduler
 
 ################################################################################
+# keys for Topology Update command
+################################################################################
+TOPOLOGY_UPDATE_DEACTIVATE_SECONDS:  heron.topology.update.deactivate.seconds
+TOPOLOGY_UPDATE_REACTIVATE_SECONDS:  heron.topology.update.reactivate.seconds
```

Comment:
No, `scheduler.yaml` doesn't use this. The current intent is that it can be set in one of two ways:

1. in the topology main code by calling `setUpdateDeactivateWaitDuration(Map<String, Object> conf, int seconds)` on `Config`.
2. By passing it as a command line arg vai `--config-property` when calling `heron update`.

If that's the case, it seems I don't need the entires in `keys.yaml`, `defaults.yaml` and `Context`.